dbt/scripts/export_models.py

```diff
@@ -198,10 +199,34 @@ def main():
                 ),
             )
             table.tableStyleInfo = TableStyleInfo(
-                name="TableStyleMedium7", showRowStripes=True
+                name="TableStyleMedium11", showRowStripes=True
```

Comment:
I felt like the look of `TableStyleMedium11` more closely matched Will's reports than `TableStyleMedium7`, but I don't really think it matters very much!